nni/contrib/compression/base/config.py

```diff
@@ -177,7 +180,7 @@ def select_modules_by_config(model: torch.nn.Module, config: Dict[str, Any]) ->
     for op_type in exclude_op_types:
         op_names.difference_update(type2names.get(op_type, set()))
 
-    return {module_name: name2module[module_name] for module_name in op_names}, config
+    return {module_name: name2module[module_name] for module_name in op_names}, config, fuse_names
```

Comment:
please search `select_modules_by_config` and check this retrun values change is legal.